subgraphs/accounts/resolvers.js

```diff
@@ -13,7 +14,18 @@ export const resolvers = {
   },
   BankAccount: {
     __resolveReference(ref) {
+      console.log('resolving BankAccount reference', ref.id);
       return getAccountById(ref.id);
     }
   },
+  Transaction: {
+    __resolveReference(ref) {
+      console.log('resolving Transaction reference', ref.id);
```

Comment:
This can be removed if the schema is removed now, but also we don't want to check in console logs to prod resolvers